### PR TITLE
Add #define BOOT_VOLUME option

### PR DESCRIPTION
**Pull request by NoSloppy**

### CODE REVIEW

File: sound/dynamic_mixer.h [hunk 1]
```diff
@@ -185,7 +185,10 @@ template<int N> class AudioDynamicMixer : public ProffieOSAudioStream, Looper {
   int32_t peak_sum_ = 0;
   int32_t peak_ = 0;
   int32_t num_samples_ = 0;
-  int32_t volume_ = VOLUME;
+#ifndef BOOT_VOLUME
```
Comment (profezzorn) [inline sound/dynamic_mixer.h:188]: The #ifdef should go in ProffieOS.ino so that BOOT_VOLUME can be used anywhere.

File: sound/dynamic_mixer.h [hunk 2]
```diff
@@ -185,7 +185,10 @@ template<int N> class AudioDynamicMixer : public ProffieOSAudioStream, Looper {
...
+  #define BOOT_VOLUME VOLUME
```
Comment (profezzorn) [inline sound/dynamic_mixer.h:189]: No indentation for #defines

File: sound/dynamic_mixer.h (hunk 1 above)
Comment (NoSloppy) [reply sound/dynamic_mixer.h]: ah. Ok done.

File: sound/dynamic_mixer.h (hunk 2 above)
Comment (NoSloppy) [reply sound/dynamic_mixer.h]: done

**Pull request merged**